src/CMakeLists.txt

```diff
@@ -315,23 +315,6 @@ if (WIN32)
         POST_BUILD
         COMMAND ${CMAKE_STRIP} "${CMAKE_BINARY_DIR}/${PROJECT_NAME}.exe")
 
-    # Code signing
```

Comment:
Rejected - but we do need a better way of specifying these paths. Code signing is not optional for Windows releases.